exchanges/deribit/deribit_wrapper.go

```diff
@@ -385,19 +351,25 @@ func (d *Deribit) UpdateOrderbook(ctx context.Context, p currency.Pair, assetTyp
 		Asset:           assetType,
 		VerifyOrderbook: d.CanVerifyOrderbook,
 	}
-	book.Asks = make([]orderbook.Item, len(obData.Asks))
+	book.Asks = []orderbook.Item{}
```

Comment:
You can define cap even if you aren't using it like this: `make([]orderbook.Item, 0, len(obData.Asks))`